packages/babel-plugin-transform-class-properties/README.md

```diff
@@ -1,16 +1,46 @@
 # babel-plugin-transform-class-properties
 
+> This plugin transforms es2015 static class properties as well as
```

Comment:
As @xtuc mentioned, due to the way we parse these we need the descriptions on a single line.